locale/exchangecalendar/en-US/rtews.dtd

```diff
@@ -1,15 +1,15 @@
- <!ENTITY rtews.selectaccountdesc "Select mail account for which to configure. The account has to be a Microsoft Exchange based account">
+ <!ENTITY rtews.selectaccountdesc "Select Microsoft Exchange mail account for which to configure.">
  <!ENTITY rtews.accconfigured "Account already configured">
- <!ENTITY rtews.dicoverdesc "You may select to autodiscover the Microsoft Exchange Server (EWS) URL or configure it manually">
+ <!ENTITY rtews.dicoverdesc "You may choose to autodiscover the Microsoft Exchange Server (EWS) URL or configure it manually">
```

Comment:
May be we should write: "Exchange Web Server (EWS)" instead of "Microsoft Exchange Server (EWS)".